templates/content/blank/.template.config/template.json

```diff
@@ -50,7 +50,7 @@
       "type": "parameter",
       "dataType": "string",
       "replaces": "FabulousAvaloniaPkgVersion",
-      "defaultValue": "PKG_VERSION"
```

Comment:
Don't forget to revert the version to `PKG_VERSION` before merging this branch. Otherwise the templates won't automatically use the latest version.

Reply:
Thanks. Will do